server/token.go

```diff
@@ -775,7 +775,7 @@ func validateCodeVerifier(verifier, challenge, method string) error {
 		return err
 	}
 
-	if generatedChallenge != challenge {
+	if subtle.ConstantTimeCompare([]byte(generatedChallenge), []byte(challenge)) != 1 {
```

Comment:
`challenge` is a SHA256 hash that can be tried only once. Do we need to use `ConstantTimeCompare` for it and add extra latency?